### PR TITLE
Feature/3017 activate edit and delete icons 

### PR DESCRIPTION
**Pull request by Harshvardhan80**
Targets #3017 

### Description
Activate the edit icon
Activate delete icon

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Left some comments. 
And As far I can see `database/migrations/2023_04_08_181613_create_office_locations_table.php` this file is empty. If it's not needed, then Please remove this file from this PR.

File: database/migrations/2023_04_17_113950_create_office_locations_table.php [hunk 1]
```diff
@@ -1,38 +0,0 @@
...
-    }
-
-    /**
-     * Reverse the migrations.
-     *
-     * @return void
-     */
-    public function down()
-    {
-        Schema::dropIfExists('office_locations');
```
Comment (Ayush8923) [inline database/migrations/2023_04_17_113950_create_office_locations_table.php:36]: Why did you delete this file?

File: Modules/Operations/Routes/web.php [hunk 2]
```diff
@@ -14,4 +14,7 @@
 Route::prefix('operations')->group(function () {
     Route::get('/office-location', 'OperationsController@index')->name('office-location.index');
     Route::post('/office-location', 'OperationsController@store')->name('office-location.store');
+    Route::get('/office-location/{id}/edit', 'OperationsController@edit')->name('office-location.edit');
+    Route::put('/office-location/{id}/update', 'OperationsController@update')->name('office-location.update');
+    Route::delete('/office-location/{id}', 'OperationsController@destroy')->name('office-location.destroy');
```
Comment (Ayush8923) [inline Modules/Operations/Routes/web.php:19]: Please update the all places.
```suggestion
    Route::delete('/office-location/{id}', 'OperationsController@delete')->name('office-location.delete');
```

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Left some comments. Please check.
And after making the suggested changes, please test locally once.

File: database/migrations/2023_04_17_113950_create_office_locations_table.php (hunk 1 above)
Comment (Harshvardhan80) [reply database/migrations/2023_04_17_113950_create_office_locations_table.php]: It was deleted by mistake i added it.

File: Modules/Operations/Routes/web.php (hunk 2 above)
Comment (Harshvardhan80) [reply Modules/Operations/Routes/web.php]: I have done.

File: Modules/Operations/Http/Controllers/OperationsController.php [hunk 3]
```diff
@@ -39,31 +39,39 @@ public function store(Request $request, OperationService $service)
...
     /**
      * Update the specified resource in storage.
      */
-    public function update()
+    public function update(Request $request, $id, OperationService $service)
     {
+        $data = $request->all();
+        $centre = $service->update($data, $id);
+
+        return redirect()->route('office-location.index');
```
Comment (Ayush8923) [inline Modules/Operations/Http/Controllers/OperationsController.php:61]: Just checking, After updating the form, is the updated data being shown on the index page?

**Review by Ayush8923 — CHANGES_REQUESTED** [review]

File: Modules/Operations/Http/Controllers/OperationsController.php (hunk 3 above)
Comment (Harshvardhan80) [reply Modules/Operations/Http/Controllers/OperationsController.php:61]: yes showing on index page

**Review by Ayush8923 — APPROVED** [review]

File: Modules/Operations/Routes/web.php [hunk 4]
```diff
@@ -14,4 +14,7 @@
 Route::prefix('operations')->group(function () {
     Route::get('/office-location', 'OperationsController@index')->name('office-location.index');
     Route::post('/office-location', 'OperationsController@store')->name('office-location.store');
+    Route::get('/office-location/{id}/edit', 'OperationsController@edit')->name('office-location.edit');
+    Route::put('/office-location/{id}/update', 'OperationsController@update')->name('office-location.update');
+    Route::delete('/office-location/{id}', 'OperationsController@delete')->name('office-location.delete');
```
Comment (GauravGusain98) [inline Modules/Operations/Routes/web.php:19]: If you use route model binding then the variable name should be same as parameter name given in controller. So you need to change the `id` to proper name based on the model object that you will get in controller

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments and read about route model binding

**Review by GauravGusain98 — APPROVED** [review]
LGTM
